shpkpr/marathon/client.py

```diff
@@ -118,7 +129,7 @@ def deploy_application(self, application, force=False):
 
         # raise an appropriate error if something went wrong
         if response.status_code == 409:
-            raise ClientError("App is locked by one or more deployments: %s" % response.json()['deployments'][0]['id'])
+            raise ClientError("App(s) locked by one or more deployments: %s" % response.json()['deployments'][0]['id'])
```

Comment:
:question: Could `response.json()['deployments']` now have more than one deployment? Maybe we need to print all the locked apps in that case.